muxpi/parse_image.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+ARCHVARIANT="$1"
+DUT_IP="$2"
+NEPTUNE_IMAGE="$3"
+
+MUXPI="$HOME/muxpi"
+IMAGES="$MUXPI/images"
```

Comment:
In my opinion these variable names aren't very descriptive. These are the paths to the muxpi and to the muxpi images, correct? The word PATH could be included in the name.

Reply:
Changed